docs/broken_links.py

```diff
@@ -0,0 +1,58 @@
+import re
```

Comment:
This utility script appears to belong under `docs/pylib/`

Reply:
It also requires a doc string, like every other python resource.

Reply:
A broken-link checker is a useful resource though! :+1:

Reply:
Not part of the docs refactor, just a tool I made for my own workflow.